src/plugins/info/plugin.py

```diff
@@ -18,20 +18,24 @@
 # FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS
 # IN THE SOFTWARE.
 
+import glob
 import json
 import logging
 import os
 import platform
 import requests
+import site
 import sys
 
+import yaml
 from colorama import Fore, Style
 from importlib.metadata import distributions, version
 from io import BytesIO
 from markdown.extensions.toc import slugify
+from mkdocs.config.defaults import MkDocsConfig
 from mkdocs.plugins import BasePlugin, event_priority
-from mkdocs.structure.files import get_files
-from mkdocs.utils import get_theme_dir
+from mkdocs.utils import get_yaml_loader
+import regex
```

Comment:
Please move this to line 26, i.e., in the top import block.

Reply:
Am I right to understand you split the import blocks into:
1. Simple `import xyz`
2. `from foo import bar`
3. `from .internal import util`
?

That's a bit unconventional as typically it's:
1. Import built-in
2. Import external installed 
3. Import internal

My `import regex` doesn't meet any criteria, a well spotted mistake, but I wanted to used to be sure.

Reply:
Yes, currently we use that convention. Please stick to it for now for consistency. We can change it in the future.